### PR TITLE
resharper

### PR DESCRIPTION
**Pull request by jonathanmcdermid**
This is just me applying as many of the resharper suggestions I can stomach, the size of the MR is primarily due to how much changes when you enforce common naming conventions. There really shouldn't be any functional changes, so if you see something that looks like one, point it out and we can discuss.

### CODE REVIEW

File: src/decoders/novatel/src/rangecmp/range_decompressor.cpp [hunk 1]
```diff
@@ -6,7 +32,7 @@ using namespace novatel::edie;
 using namespace novatel::edie::oem;
 
 //-----------------------------------------------------------------------
-//! Table used to expand the scaled Pseudorange STDs.  This is defined
+//! Table used to expand the scaled pseudo range STDs.  This is defined
```
Comment (zainarnoldhexagon) [inline src/decoders/novatel/src/rangecmp/range_decompressor.cpp:35]: not super important, but I think pseudorange is the correct term here.

**Review by zainarnoldhexagon — COMMENTED** [review]
I didn't see anything out of the ordinary! I would approve, but I don't know if that auto-merges your changes, so when you've got eyes on this from other people and you're ready, just ping me!

File: src/decoders/novatel/src/rangecmp/range_decompressor.cpp (hunk 1 above)
Comment (jonathanmcdermid) [reply src/decoders/novatel/src/rangecmp/range_decompressor.cpp]: fixed!

**Comment by Brian-Pho** [comment]
Good changes here

**Pull request merged**